resources/api/openapi.yml

```diff
@@ -173,23 +179,49 @@ components:
         name:
           type: string
           example: Heaven
+        description:
+          type: string
+          example: Located behind room 42
+          description: Description of the room
         url:
           type: string
           example: https://example.com/location/42
           description: Link of the location page
       required:
         - id
         - name
+        - description
         - url
+    NeededAngelType:
+      type: object
+      properties:
+        entries:
+          type: array
+          items:
+            $ref: '#/components/schemas/ShiftEntry'
+        angel_type:
+          $ref: '#/components/schemas/Reference'
+        needs:
+          type: integer
+          description: >
+            Number of users needed for the shift of the given type.
+            Will be more than users count when not full, less when overbooked
+            or 0 when only additional users with given type have been added.
+          example: 3
+      required:
+        - entries
+        - angel_type
+        - needs
     Shift:
       type: object
       properties:
         id:
           type: integer
           example: 42
-        title:
+        name:
           type: string
           example: Cleanup the venue
+          description: Title of the shift
         description:
```

Comment:
<img width="1000" alt="Screenshot 2024-06-09 at 18 42 18" src="https://github.com/engelsystem/engelsystem/assets/73076134/9543ead2-70de-400a-9c64-51018b05267a">
<img width="690" alt="Screenshot 2024-06-09 at 18 42 31" src="https://github.com/engelsystem/engelsystem/assets/73076134/9ad51820-6ebc-4ea6-ba23-28b0428f228b">

the description does not work?

Reply:
<img width="1347" alt="Screenshot 2024-06-09 at 19 02 00" src="https://github.com/engelsystem/engelsystem/assets/73076134/69b45e9b-497b-4644-ba76-ea77a179f919">
I mean this description 
and please fix the new line

Reply:
> Shift description, should be added to the shift type description but might be empty.
> Normally contains additional information for a specific shift / task.